### DISCUSSION TITLE
Where to download the full tailwind.css

### DISCUSSION THREAD

**Discussion by michealp-coder** [tailwindlabs/tailwindcss#5414] [Help] [answered]
Where can I download the full/complete tailwind.css file (non-compressed and non-purged)?

I'd like to do some diff analysis (to assist on https://github.com/tailwindlabs/tailwindcss/discussions/5400).

**Comment by NovusOpus** [comment, marked as answer]
You should start by reading this in the docs :https://tailwindcss.com/docs/installation

The [CDN](https://tailwindcss.com/docs/installation#using-tailwind-via-cdn) part includes the FULL non-purged css file, but that file is not going to include 'variants' and some other stuff that you can see listed in the [CDN Documentation](https://tailwindcss.com/docs/installation#using-tailwind-via-cdn).

The file is compressed but you can use a plugin in VS Code or an online CSS Beautify tool to make the CSS file readable.

**Comment by michealp-coder** [comment]
Thanks so much for the help

**Comment by PiengBol** [comment]
I'm also looking for offline version.

**Reply by blip-cmd** [reply]
https://tailwindcss.com/docs/installation
